src/main/java/ReviewResponse.java

```diff
@@ -13,6 +16,8 @@ public class ReviewResponse {
     @Getter
     String okbb;
 
+    @Setter
+    String blabla;
 
```

Comment:
:information_source: Method name 'newMethod' is not descriptive. (Severity: Minor)